src/lib/core/style/_noop-animation.scss

```diff
@@ -0,0 +1,9 @@
+// If the mat-animation-noop class is present on the components root element,
+// prevent non css animations from running.
+@mixin noop-animation() {
+  @at-root .mat-animation-noop#{&} {
```

Comment:
Shouldn't this one be wrapped by the `at-root`?

```scss
@at-root {
  .mat-animation-noop {
    ...
   }
}
```

Reply:
If you do 
```
@at-root {
  .mat-animation-noop {
    ...
   }
}
```

It ends up as
`root-element-thing .mat-animation-noop`, but we want `root-element-thing.mat-animation-noop`